### DISCUSSION TITLE
Outlook opens the verification links so that users receive errors

### DISCUSSION THREAD

**Discussion by fbaeumer** [danny-avila/LibreChat#3097] [Troubleshooting] [answered]
We receive feedback that Outlook opens the verification emails and thus activates the accounts. The problem is that customers can then no longer activate the account themselves and therefore receive an error - even though the account is activated.

Is there a way to prevent Outlook from doing this with a clever email design?

**Comment by berry-13** [comment]
interesting, I'll try to test the verification on Outlook

**Reply by fbaeumer** [reply]
You can find more information on how to prevent Outlook.com / Outlook Clients from following links in emails here: https://stackoverflow.com/questions/32851044/how-do-i-stop-outlook-com-from-following-links-in-email.

One solution is to redesign the verification page by adding a button for users to click instead of directly verifying the address. This prevents email clients from automatically opening the links and verifying the address.

**Reply by berry-13** [reply, marked as answer]
@fbaeumer I wasn't able to reproduce this bug. I created an @outlook.com email, and I can verify it without any issues

**Reply by danny-avila** [reply]
@fbaeumer I will think about how to better handle this, thanks for reporting this.
